runtime/jcl/common/sun_misc_Unsafe.cpp

```diff
@@ -96,19 +95,72 @@ Java_sun_misc_Unsafe_defineAnonymousClass(JNIEnv *env, jobject receiver, jclass
 		hostClassLoader = vm->systemClassLoader->classLoaderObject;
 	}
 	jobject hostClassLoaderLocalRef = vmFuncs->j9jni_createLocalRef(env, hostClassLoader);
+
+	J9ClassPatchMap cpPatchMap = {0, NULL};
+	j9array_t patchArray = NULL;
+	PORT_ACCESS_FROM_ENV(env);
+
+	U_16 indexMap[BUFFER_SIZE];
+	if (constPatches != NULL) {
+		patchArray = (j9array_t)J9_JNI_UNWRAP_REFERENCE(constPatches);
+		cpPatchMap.size = (U_16)J9INDEXABLEOBJECT_SIZE(currentThread, patchArray);
+		if (cpPatchMap.size <= BUFFER_SIZE) {
+			cpPatchMap.indexMap = indexMap;
+		} else {
+			cpPatchMap.indexMap = (U_16 *)j9mem_allocate_memory(cpPatchMap.size * sizeof(U_16), J9MEM_CATEGORY_VM);
+
+			if (cpPatchMap.indexMap == NULL) {
+				vmFuncs->throwNativeOOMError(env, 0, 0);
```

Comment:
This function gets VMAccess so it can't be called with it.  Need to swap these two lines